lib/pronto/formatter/text_formatter.rb

```diff
@@ -1,10 +1,26 @@
 module Pronto
   module Formatter
     class TextFormatter
+      include Colorizable
+
+      LOCATION_COLOR = :cyan
+
+      LEVEL_COLORS = {
+        info: :yellow,
+        warning: :magenta,
+        error: :red,
+        fatal: :red
+      }.freeze
+
+      attr_reader :config
+
+      def initialize(config = Config.new)
```

Comment:
If this isn't constructed with a config outside of tests, how would I get a configured value other than the default of false?

I think theres a missing argument to the constructor somewhere that passes in the config, right?